lib/pkg.mli

```diff
@@ -169,6 +169,12 @@ val version_line_re : Re.t
 val prepare_opam_for_distrib :
   version:string -> content:string list -> string list
 
+(** {1 Dune project} *)
+
+val dune_project_name : Fpath.t -> (string option, [> Rresult.R.msg ]) result
+(** Returns the name stanza entry of the dune project, if any; else, returns
+    `None`. Returns an error, if one of the system calls fails. *)
```

Comment:
I don't really like exposing even more functions in `Pkg`, particularly one that doesn't even depend on some `Pkg.t`. What do you two think? Is that ok or would you move that somewhere else?

Reply:
I thought there already was a library to read dune-project files, well I looked for it but couldn't find anything, maybe it's planned to get it out of dune but wasn't done yet

Reply:
Do you mean dune itself exposing its library to parse the dune project or an independent library?

Reply:
I was thinking of an independent library

Reply:
Maybe it can make sense to put this function in a dune_project.ml file, so we can put all similar functions there if ever we do more checks on dune-project files in the future

Reply:
This fonction will be useful in the future I think. We should split `Pkg` into `Pkg` and `Repo`. It could fit the latter.

In the meantime I think it's okay to have it here!